CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## [1.0.0-rc.3](https://github.com/commercetools/merchant-center-application-kit/compare/v1.0.0-rc.2...v1.0.0-rc.3) (2018-11-05)
+
+#### 🐛 Type: Bug
+
+- `mc-http-server`
+  - [#27](https://github.com/commercetools/merchant-center-application-kit/pull/27) refactor(http-server): configure cloudbuilder to wait for 30s before building the docker image ([@emmenko](https://github.com/emmenko))
+- `react-notifications`
+  - [#48](https://github.com/commercetools/merchant-center-application-kit/pull/48) fix(components/notification-kinds): do not report error to sentry for `invalid_scope` ([@adnasa](https://github.com/adnasa))
```

Comment:
This was manually added, since I didn't label my PR properly 🤕

Reply:
It's fine to add things manually when drafting a release.